### PR TITLE
Recode TikTok presence

### LINKED ISSUE #4422: TikTok | tiktok.com
**Describe the Bug**
. --> viewing undefined even im not when watching tiktok,and the link to watch that video appears in discord is wrong too
**Steps to Reproduce**
1.I need a video to help me fix this bug or an unambiguous set of steps to reproduce this bug.
**Expected Behavior**                                                                                                                                                                           
- Required, describe what was expected to happen.                                                                                      
**Screenshots**
im sorry i dont know how to add a screenshot here
**Environment**
-  Windows 10
- Chrome
- Version: 80.0.334.4 (64-bit) 

**Additional context**
Thanks

### PR DESCRIPTION
**Pull request by TheGamerzs**
image
image
image

### CODE REVIEW

File: websites/T/TikTok/presence.ts [hunk 1]
```diff
@@ -1,133 +1,105 @@
...
-  following: string;
-  buttonViewProfile: string;
-  viewProfile: string;
-  viewTikTok: string;
-  buttonViewTikTok: string;
-}
-
 const presence = new Presence({
-    clientId: "809093093600133165"
+    clientId: "893257746877644830"
```
Comment (Bas950) [inline websites/T/TikTok/presence.ts:2]: ???

File: websites/T/TikTok/presence.ts [hunk 2]
```diff
@@ -1,133 +1,105 @@
...
-  oldLang: string = null;
+  strings = presence.getStrings({
+    browse: "general.browsing",
+    forYou: "tiktok.forYou",
+    following: "tiktok.following",
+    buttonViewProfile: "general.buttonViewProfile",
+    viewProfile: "general.viewProfile",
+    viewTikTok: "tiktok.viewing",
+    buttonViewTikTok: "tiktok.buttonViewTikTok"
+  }),
```
Comment (Bas950) [inline websites/T/TikTok/presence.ts:12]: not using the language from the settings?

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/T/TikTok/presence.ts (hunk 1 above)
Comment (TheGamerzs) [reply websites/T/TikTok/presence.ts:2]: Needed more assets

Comment (Bas950) [reply websites/T/TikTok/presence.ts:2]: then just ask me to add them to it?

Comment (TheGamerzs) [reply websites/T/TikTok/presence.ts]: Turns out the asset is already there

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**